src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -254,6 +256,25 @@ private Step CreateStep(Step jsonStepInfo, HostControlInfo hostControlInfo, int
         private void Popup_StepClosed(string name, Step.StepTypes stepType)
         {
             GuideFlowEvents.OnGuidedTourFinish(currentGuide.Name);
+
+            //The exit tour popup will be shown only when a popup (doesn't apply for survey) is closed or when the tour is closed. 
+            if(stepType != Step.StepTypes.SURVEY)
+                CreateRealTimeInfoWindow();
+        }
+
+        private void CreateRealTimeInfoWindow()
+        {
+            //Search a UIElement with the Name "statusBarPanel" inside the Dynamo VisualTree
+            UIElement hostUIElement = Guide.FindChild(mainRootElement, "statusBarPanel");
+
+            //Creates the RealTimeInfoWindow popup and set up all the needed values to show the popup over the Dynamo workspace
+            var exitTourPopup = new RealTimeInfoWindow();
+            exitTourPopup.VerticalOffset = 30;
```

Comment:
Does this define the offset between the RealTimeInfoWindow with the boarder of Dynamo workspace?